zeeguu/core/elastic/elastic_query_builder.py

```diff
@@ -182,3 +184,64 @@ def build_elastic_search_query(
     query = {"size": count, "query": weighted_query.to_dict()}
 
     return query
+
+def build_elastic_more_like_this_query(
+    language: Language,
+    like_documents: list[dict[str, str]],
+    similar_to: list[str],
+    cutoff_days: int,
+    scale: str ="10d",
+    offset: str ="4h",
+    decay: float=0.9):
+    
+    cutoff_date = datetime.now() - timedelta(days=cutoff_days)
+
+    query = {
```

Comment:
json is not for humans !!! 
in other queries i built little python DSLs that help build up such a monster w/o having to dump the whole string in the code. i'll accept it like this for now, but if you find the time after you've defended, i'd be happy to receive an updated code version that is less a monster string and more a nicely constructed python expression.

Reply:
I'll make sure to take a look at it again and extend the DSL with what I need and then build the same query with it.
Thanks for the feedback